hazelcast/src/main/java/com/hazelcast/sql/impl/SqlDataSerializerHook.java

```diff
@@ -123,7 +129,14 @@
     public static final int EXPRESSION_IS_NOT_FALSE = 39;
     public static final int EXPRESSION_IS_NOT_NULL = 40;
 
-    public static final int LEN = EXPRESSION_IS_NOT_NULL + 1;
+    public static final int EXPRESSION_ABS = 41;
+    public static final int EXPRESSION_SIGN = 42;
+    public static final int EXPRESSION_RAND = 43;
+    public static final int EXPRESSION_DOUBLE = 44;
+    public static final int EXPRESSION_FLOOR_CEIL = 45;
+    public static final int EXPRESSION_ROUND_TRUNCATE = 46;
```

Comment:
Might be better to name these `FUNCTION_x`

Reply:
We name them this way because the relevant classes all extend `com.hazelcast.sql.impl.expression.Expression` class.